ru.json

```
@@ -0,0 +1,181 @@
+{
+	"footer": {
+		"description": "Modrinth — программное обеспечение с открытым кодом.",
+		"copyright": "© Guavy LLC",
+		"legal": {
+			"heading": "Legal",
+			"terms": "Условия использования",
+			"privacy": "Политика конфиденацильности",
+			"rules": "Правила"
+		},
+		"resources": {
+			"heading": "Ресурсы",
+			"blog": "Блог",
+			"docs": "Документация"
+		},
+		"buttons": {
+			"privacy": "Настройки конфиденциальности",
+			"theme": {
+				"label": "Изменить тему",
+				"system": "Системная тема",
+				"light": "Светлая тема",
+				"dark": "Тёмная тема",
+				"oled": "OLED тема"
+			},
+			"language": {
+				"label": "Выберите язык",
+				"en": "Английский",
+				"en-GB": "Английский (Соединённое Королевство)",
+				"ru-RU": "Русский (Россия)"
+			}
+		}
+	},
+	"error": {
+		"status": "Ошибка {status}",
+		"message": {
+			"404": "Ух ох, похоже вы заблудились в [лабиринте](https://github.com/modrinth/labrinth).<br/>Страница, которую вы ищете, не найдена.",
```

```suggestion
			"404": "Ой-йо, похоже вы заблудились в лабиринте.<br/>Страница, которую вы ищете, не найдена.",
```
Надо бы поставить другие междометия, например `Ой-йо`, `Ого` или `Ох`.

---

Ой-ёй может быть? Я не слышал чтобы кто-то говорил «ой-йо» в таком ключе, только «ой-йо, ты погляди что он творит».

---

Я не нашёл литературных примеров использования `Ой-ёй`

---

Reverso Context знает об этом варианте: https://context.reverso.net/translation/russian-english/%D0%9E%D0%B9-%D1%91%D0%B9.

---

ну тогда хорошо